src/modules/job-manager/alloc.c

```diff
@@ -81,10 +81,6 @@ static void interface_teardown (struct alloc *alloc, char *s, int errnum)
                 job->alloc_queued = 1;
```

Comment:
Another on the commit message, how about "job-manager: stop publishing job-annotations message"

The use of "event" gets mighty confusing.